lib/matcher.js

```diff
@@ -26,13 +26,13 @@ Matcher.prototype.setName = function(name) {
   this.name = name;
 };
 
-Matcher.prototype.match = function(path, value) {
+Matcher.prototype.match = function(path, value, index) {
```

Comment:
I believe this change is breaking the strummer-middleware: https://github.com/Tabcorp/strummer-middleware/blob/94a0eb96c04b6ecb404c76cbdc90af6ebabf5c4d/index.js#L62 Specifically this check. Should we update the middleware as well @deancouch ?